Sources/CartonHelpers/Parsers/DiagnosticsParser.swift

```diff
@@ -162,9 +162,9 @@ public struct DiagnosticsParser: ProcessOutputParser {
       var groupedMessages = [[CustomDiagnostic]]()
       for message in messages {
         if let lastLineStr = groupedMessages.last?.last?.line,
-           let lastLine = Int(lastLineStr),
-           let line = Int(message.line),
-           lastLine == line - 1 || lastLine == line
+          let lastLine = Int(lastLineStr),
+          let line = Int(message.line),
+          lastLine == line - 1 || lastLine == line
```

Comment:
Are you on the latest version of SwiftFormat? I suspect this part of the diff will go away if you update.